### ISSUE TITLE
Variable type not showing in hover window

### ISSUE THREAD

**Issue by scallaway** [pappasam/jedi-language-server#295]
I appreciate there has been a load of work done surrounding the hover window etc. over the years, but I think there's one last bit that's missing.

Currently, the information I'm seeing in the hover window (based on the JLS source code) is the following:
![image](https://github.com/pappasam/jedi-language-server/assets/16577635/14b896fe-ad89-4f8f-9535-2316c18a7788)

I think what's crucially missing is the actual _type_ of `markup_kind` in that context, as a result of calling `_choose_markup()`. I initially thought this information was stored in the "Full Name" field, but realised that's just the drill-down location of the variable.

![image](https://github.com/pappasam/jedi-language-server/assets/16577635/6db65583-3387-4125-94d7-c85d6281ba3b)

Calling hover on `_choose_markup()` does indeed show `def _choose_markup(server: JediLanguageServer) -> MarkupKind`, therefore I find it reasonable to expect `markup_kind` to show that it's of type `MarkupKind`.

Basically, I'd expect to see the following in the dropdown window for `markup_kind`:
```
markup_kind [MarkupKind] = _choose_markup(server)
```

I have played around with the configuration, as well as going through the documentation, but haven't been able to come to a good solution for this.

I'm not 100% sure whether this is a bug or just a config issue, but any help on this would be greatly appreicated!

**Comment by scallaway** [comment]
This has been open for a while - I've just had another play around with Jedi again and it still shows the same information.

Any chance someone can have a look at this (or provide some insight)?
